src/pybind/mgr/snap_schedule/fs/schedule.py

```diff
@@ -397,7 +397,7 @@ def parse_schedule(cls, schedule: str) -> Tuple[int, str]:
     @property
     def repeat(self) -> int:
         period, mult = self.parse_schedule(self.schedule)
-        if mult == 'M':
+        if mult == 'm':
```

Comment:
nit: explain a bit in the commit message.

Reply:
added reasoning for commit in the commit message